frontend/openapitools.json

```diff
@@ -7,7 +7,7 @@
       "fetch": {
         "generatorName": "typescript-fetch",
         "output": "src/generated-sources",
-        "glob": "../contract/src/main/resources/swagger/kafbat-ui-api.yaml",
+        "glob": "../contract-typespec/build/tsp/api/openapi.yaml",
```

Comment:
do we deliberately use typespec (via gradle's useTypeSpec) conditionally on the backend and exclusively on the frontend rn?